var/spack/repos/spack_repo/builtin/packages/amr_wind/package.py

```diff
@@ -24,6 +24,9 @@ class AmrWind(CMakePackage, CudaPackage, ROCmPackage):
     license("BSD-3-Clause")
 
     version("main", branch="main", submodules=True)
+    version(
+        "3.5.0", tag="v3.5.0", commit="TBD", submodules=True
```

Comment:
It's best to leave the commit out if you don't actually have the sha pinned down yet. This will soon lead to concretization errors as we will be ingesting and assigning commit sha's to installs soon.

Reply:
Sorry Phil, I made this PR to illustrate the plan -- I should have marked it as draft from the start. I'll make sure all of the versions and commits are real and listed before I mark it as ready for review.